src/site/_data/postTags.js

```diff
@@ -185,6 +185,9 @@ const tags = {
   'web-bundles': {
     title: 'Web Bundles',
   },
+  'web-vitals': {
+    title: 'Web Vitals',
+  },
```

Comment:
@robdodson || @samthor need your review b/c of proposed change to add a tag for `Web Vitals`